apps/survey/contracts/Survey.sol

```diff
@@ -253,11 +253,11 @@ contract Survey is AragonApp {
             bool _open,
             address _creator,
             uint64 _startDate,
-            uint256 _snapshotBlock,
-            uint256 _minParticipationPct,
+            uint64 _snapshotBlock,
+            uint64 _minParticipationPct,
+            uint64 _options,
             uint256 _votingPower,
-            uint256 _participation,
-            uint256 _options
+            uint256 _participation
```

Comment:
Let's revert the return order; they're already not in sync with the storage struct anyway, and changing it means we'd have to change it in the frontend as well.

Reply:
Apologies, not sure why I made that change.